edge/pkg/metamanager/process.go

```diff
@@ -461,6 +461,7 @@ func (m *metaManager) processDelete(message model.Message) {
 }
 
 func (m *metaManager) processQuery(message model.Message) {
+	//dao.DeleteMetaByType("mission")
```

Comment:
remove this?

Reply:
Ah. Thanks for catching this. It is a leftover from debugging.